odysseus/HeraclesQuest.ts

```diff
@@ -199,9 +209,7 @@ export type HeraclesQuestTask = {
     amount?: number;
 } | {
     type: 'heracles:location';
-    icon?: ResourceLocation;
-    title: Component;
-    description: Component;
+    description: string;
```

Comment:
Any reason this is converted to `string`?

Reply:
uhh this is mirroring me messing with it on the heracles end. Because everything has a title and icon override, and they use strings, the buggy use of component/text got changed.